recipe/cgal-mpir.patch

```diff
@@ -77,24 +77,33 @@ index b5e943e..660d3c1 100644
 --- Number_types/include/CGAL/GMP/Gmpzf_type.h
 +++ Number_types/include/CGAL/GMP/Gmpzf_type.h
 @@ -82,7 +82,7 @@ public:
- 
+
    // exponent type
    // --------------------
 -  typedef long Exponent; // may overflow, but if it does, the mantissa is
-+  typedef signed long Exponent; // may overflow, but if it does, the mantissa is
++  typedef mp_size_t Exponent; // may overflow, but if it does, the mantissa is
```

Comment:
@edisongustavo shouldn't this be mp_exp_t ?
https://gmplib.org/manual/Nomenclature-and-Types.html